store-rx2/src/main/kotlin/com/dropbox/store/rx2/RxFetcher.kt

```diff
@@ -19,9 +18,9 @@ import kotlinx.coroutines.reactive.asFlow
  *
  * @param flowableFactory a factory for a [Flowable] source of network records.
  */
-fun <Key : Any, Output : Any> flowableFetcher(
+fun <Key : Any, Output : Any> Fetcher.Companion.fromFlowableFetcher(
```

Comment:
we can drop the `Fetcher` suffix now: `Fetcher.fromFlowable` is descriptive enough